bundles/framework/publisher2/resources/scss/style.scss

```diff
@@ -37,11 +37,34 @@ div.notLoggedIn {
     padding: 10px;
 }
 
+div.publisher-announcements-inputs {
+    display: grid;
+    grid-column-gap: 50px;
+    grid-template-columns: auto auto;
+
+    .icon-warning-light {
+        float: right;
+        margin-left: 12px;
+    }
+}
+
 div.basic_publisher {
     // Temp, change inputs to TextInput and get rid of this
     input[type=text] {
         width: 260px;
     }
+    
+    .publisher-layout-announcements {
+        padding-left: 28px;
+      
+        input[name=publisher-announcements] {
+            width: 125px;
+    
+            button {
+                float: right;
+            }
+        }
+    }
```

Comment:
Move these scss-definitions to admin-announcements where the announcement tool is defined.